R/biokinetics.R

```diff
@@ -142,6 +142,11 @@ biokinetics <- R6::R6Class(
 
       dt_out[, t_id := NULL]
 
+      if (private$scale == "natural") {
```

Comment:
brought this transformation forwards so that it is always applied before aggregating

https://github.com/seroanalytics/epikinetics/pull/23/files#diff-ba4d2548084653348f5c9eab3d79243a275dd7be9a823c403b4af7330f7ee8ceL395